src/core/mixins/handlerWallet.mixin.js

```diff
@@ -0,0 +1,184 @@
+        }
+      }
+    },
+    /**
+     * Apollo query to fetch latest tokens
+     * also set eth price for state
+     */
+    getLatestPrices: {
+      query: getLatestPrices,
+      skip() {
+        return !this.isEthNetwork;
+      },
+      result({ data }) {
+        this.tokensData = new Map();
+        if (data && data.getLatestPrices) {
+          data.getLatestPrices.forEach(token => {
+            const isEth = token.id === tokens.eth;
+            if (isEth || token.contract) {
+              if (isEth) {
+                const usd = {
+                  value: token.current_price,
+                  symbol: '$',
+                  name: 'USD',
+                  price_change_24h: token.price_change_24h
+                };
+                this.setETHUSDValue(usd);
+              }
+              this.tokensData.set(
+                token.contract ? token.contract.toLowerCase() : tokens.eth,
+                token
+              );
+            }
+          });
+        }
+      },
+      error(error) {
+        Toast(error.message, {}, ERROR);
+      }
+    },
+    /**
+     * Apollo query to fetch owners erc20 tokens and set it to state
+     */
+    getOwnersERC20Tokens: {
+      query: getOwnersERC20Tokens,
+      variables() {
+        return {
+          hash: this.address
+        };
+      },
+      skip() {
+        return !this.address || this.address === '' || !this.isEthNetwork;
+      },
+      result({ data }) {
+        if (data && data.getOwnersERC20Tokens) {
+          const formattedList = [];
+          const ownedTokens = data.getOwnersERC20Tokens.owners;
+          ownedTokens.forEach(token => {
+            let foundToken;
+            if (this.tokensData && this.tokensData.get) {
+              foundToken = this.tokensData.get(
+                token.tokenInfo.contract.toLowerCase()
+              );
+            }
+            const denominator = new BigNumber(10).pow(token.tokenInfo.decimals);
+            const usdBalance = foundToken
+              ? new BigNumber(token.balance)
+                  .div(denominator)
+                  .times(foundToken.current_price)
+                  .toString()
+              : null;
+            const price = foundToken ? foundToken.current_price : null;
+            // need to eventually change image to check tokens network rather than just use eth network (if theres no image from coingecko)
+            formattedList.push({
+              name: token.tokenInfo.symbol,
+              symbol: token.tokenInfo.symbol,
+              subtext: token.tokenInfo.name,
+              value: token.tokenInfo.name,
+              balance: token.balance,
+              contract: token.tokenInfo.contract,
+              img: foundToken ? foundToken.image : ethImg,
+              decimals: token.tokenInfo.decimals,
+              market_cap: foundToken ? foundToken.market_cap : null,
+              price_change_24h: foundToken ? foundToken.price_change_24h : null,
```

Comment:
change this to price_change_percentage_24h

Reply:
done!